src/clarity-angular/layout/tabs/tab.ts

```diff
@@ -20,10 +21,19 @@ import {
     providers: [ IF_ACTIVE_ID_PROVIDER ]
 })
 export class Tab {
-    @ContentChild(TabLink) tabLink: TabLink;
+    @ContentChild(TabLinkDirective) tabLink: TabLinkDirective;
     @ContentChild(TabContent) tabContent: TabContent;
 
-    constructor(public ifActiveService: IfActiveService, @Inject(IF_ACTIVE_ID) public self: number) {
+    constructor(public ifActiveService: IfActiveService, @Inject(IF_ACTIVE_ID) public id: number,
+                private tabsService: TabsService) {
+
+        tabsService.register(this);
+
+        // if there's no active tab, set itself as active; it will be overridden if
+        // a tab created after this one sets it explicitly
+        if (!this.ifActiveService.current) {
+            this.ifActiveService.current = id;
+        }
```

Comment:
Do you mind adding a comment to this saying it might be worth moving to the `IfActiveService` itself? If all of our "if active" components want the first item to be active by default, we'll end up with this small piece of code duplicated everywhere.

You don't need to move it right now, just adding a comment is good enough so that people won't just copy-paste blindly but instead refactor slightly if they are doing the same thing.